fluentd/run.sh

```diff
@@ -31,6 +31,10 @@ if [ "${USE_JOURNAL:-}" != false ] ; then
     fi
 fi
 
+IPADDR4=`/usr/sbin/ip -4 addr show dev eth0 | grep inet | sed -e "s/[ \t]*inet \([0-9.]*\).*/\1/"`
```

Comment:
So `ipaddr4` and `ipaddr6` are for the node?  Do we need that?  If so, we will have to figure out a way to get that from the node, and store the container ips in the pipeline_metadata.collector (or remove them).